catalog/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+from catalog.apps import CatalogConfig
+from catalog.views import home, contacts
+
+app_name = CatalogConfig.name
+
+urlpatterns = [
+    path('home/', home, name='home'),
```

Comment:
Главную страницу лучше делать в корне, чтобы при открытии сайта она сразу открывалась и не нужно было никуда переходить
```suggestion
    path('', home, name='home'),
```